common/include/pcl/point_types_conversion.h

```diff
@@ -43,6 +43,8 @@
 #include <pcl/point_cloud.h>
 #include <pcl/point_types.h>
 
+#include <pcl/common/colors.h>
```

Comment:
Would be great if you could also add here why it is included (probably `for RGB2sRGB_LUT`?)

Reply:
Sure, I'd add an inline comment for it. Thank you!